riakcs/tests/test_riakcs.py

```diff
@@ -0,0 +1,56 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under Simplified BSD License (see LICENSE)
+
+import logging
+import pytest
+
+from socket import error as SocketError
+
+from datadog_checks.riakcs import RiakCs
+
+from . import common
+
+log = logging.getLogger(__file__)
+
+
+def test_parser(mocked_check):
+    input_json = common.read_fixture('riakcs_in.json')
+    output_python = common.read_fixture('riakcs_out.python')
+    assert mocked_check.load_json(input_json) == eval(output_python)
+
+
+def test_metrics(mocked_check, aggregator):
+    mocked_check.check(common.CONFIG)
+    expected = eval(common.read_fixture('riakcs_metrics.python'))
```

Comment:
Let's remove all `eval` and use `json.loads`

Reply:
it's not json

Reply:
It's just a dumped python object